### PR TITLE
Internal: Convert GitHub issues templates to the YAML format [ED-6117]

### PR DESCRIPTION
**Pull request by YitzhakAvm**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+name: "🐞 Bug report"
+description: "Report a bug if something isn't working as expected in the Elementor"
```
Comment (StyleShit) [inline .github/ISSUE_TEMPLATE/bug.yaml:2]:
```suggestion
description: "Report a bug if something isn't working as expected in Elementor"
```

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## BEFORE POSTING YOUR ISSUE
+
+        - Please create GitHub issues only for bugs and feature requests. GitHub issues ARE NOT FOR SUPPORT!
+
+        - If you have questions or need general support,  Please use:  https://wordpress.org/support/plugin/elementor
```
Comment (StyleShit) [inline .github/ISSUE_TEMPLATE/bug.yaml:13]:
```suggestion
        - If you have questions or need general support, please use:  https://wordpress.org/support/plugin/elementor
```

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+        - For help and support from the Elementor community, see: https://www.facebook.com/groups/Elementors/
+
+        - To read more about Elementor, check out our documentation: https://docs.elementor.com
+
+        - Developers docs are located at https://developers.elementor.com/
+
+        ### Guidelines
+
+        - Search this repository (top of the page) for the issue, and make sure it has not been fixed or reported before.
```
Comment (StyleShit) [inline .github/ISSUE_TEMPLATE/bug.yaml:23]: Maybe?

```suggestion
        - Search this repository (top of the page) for the issue, and make sure it has not been fixed or reported already.
```

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+        - Make sure you are using the most updated versions of Elementor, WordPress & your theme.
+
+        - Deactivate all of your plugins. If this solves the problem, gradually activate your plugins one by one, until you spot the problematic plugin.
+
+        - Change your WordPress theme to WordPress TwentySixteen (or other default WordPress theme).
```
Comment (StyleShit) [inline .github/ISSUE_TEMPLATE/bug.yaml:29]:
```suggestion
        - Change your WordPress theme to WordPress TwentyTwentyTwo (or other default WordPress theme).
```

File: .github/ISSUE_TEMPLATE/bug_dev_edition.yaml [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+labels: [bug,dev-edition]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## BEFORE POSTING YOUR ISSUE
+
+        - Please create GitHub issues only for bugs and feature requests. GitHub issues ARE NOT FOR SUPPORT!
+
+        - If you have questions or need general support,  Please use:  https://wordpress.org/support/plugin/elementor
```
Comment (StyleShit) [inline .github/ISSUE_TEMPLATE/bug_dev_edition.yaml:13]:
```suggestion
        - If you have questions or need general support, please use:  https://wordpress.org/support/plugin/elementor
```

Plus same as above regarding other things

File: .github/ISSUE_TEMPLATE/bug.yaml [hunk 6]
```diff
@@ -0,0 +1,78 @@
+name: "🐞 Bug report"
```
Comment (StyleShit) [inline .github/ISSUE_TEMPLATE/bug.yaml:1]: Should the extension be `YAML` or `YML`?

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 7]
```diff
@@ -0,0 +1,49 @@
+name: "🆕 Feature request"
+description: "Suggest an idea for this project"
+title: "🆕 Feature request: "
+labels: [feature]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before opening a new issue, please search for duplicate issues to prevent opening a duplicate feature request.
```
Comment (StyleShit) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:9]: IDK why, but something sounds off with this sentence ("duplicate duplicate") 😅

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+        ## The Feature
+  - type: checkboxes
+    id: prerequisites
+    attributes:
+      label: "Prerequisites"
+      options:
+        - label: "I have searched for similar features requests in both open and closed tickets and cannot find a duplicate."
+          required: true
+        - label: "The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )"
```
Comment (StyleShit) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:20]:
```suggestion
        - label: "The feature is still missing in the latest stable version of Elementor (Elementor Pro)"
```

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+          required: true
+  - type: textarea
+    id: the_problem
+    validations:
+      required: true
+    attributes:
+      label: "What problem is your feature request going to solve? Please describe."
+      description: "A clear and concise description of what the problem is."
+      placeholder: "I'm always frustrated when ..."
```
Comment (StyleShit) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:29]:
```suggestion
      placeholder: "I'm always frustrated when..."
```

Also other places (I think... correct me if I'm wrong)

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug.yaml (hunk 1 above)
Comment (ItzhakAvm) [reply .github/ISSUE_TEMPLATE/bug.yaml]: Almost all texts were copied from the current issues templates.
Anyway, fixed :D

File: .github/ISSUE_TEMPLATE/bug.yaml (hunk 2 above)
Comment (ItzhakAvm) [reply .github/ISSUE_TEMPLATE/bug.yaml]: Done.

File: .github/ISSUE_TEMPLATE/bug.yaml (hunk 3 above)
Comment (ItzhakAvm) [reply .github/ISSUE_TEMPLATE/bug.yaml:23]: I think `before` is more appropriate here.

File: .github/ISSUE_TEMPLATE/bug.yaml (hunk 4 above)
Comment (ItzhakAvm) [reply .github/ISSUE_TEMPLATE/bug.yaml]: Done.

File: .github/ISSUE_TEMPLATE/bug_dev_edition.yaml (hunk 5 above)
Comment (ItzhakAvm) [reply .github/ISSUE_TEMPLATE/bug_dev_edition.yaml]: Done, also in `custom.yaml`.

File: .github/ISSUE_TEMPLATE/feature_request.yaml (hunk 7 above)
Comment (ItzhakAvm) [reply .github/ISSUE_TEMPLATE/feature_request.yaml]: Haha definitely, as the one who wrote this in the past.
Done.

File: .github/ISSUE_TEMPLATE/feature_request.yaml (hunk 8 above)
Comment (ItzhakAvm) [reply .github/ISSUE_TEMPLATE/feature_request.yaml]: Done.

File: .github/ISSUE_TEMPLATE/feature_request.yaml (hunk 9 above)
Comment (ItzhakAvm) [reply .github/ISSUE_TEMPLATE/feature_request.yaml:29]: Also, regarding this, check the reference repo in the task description.

File: .github/ISSUE_TEMPLATE/bug.yaml (hunk 1 above)
Comment (StyleShit) [reply .github/ISSUE_TEMPLATE/bug.yaml]: Yeah I know, it just hurt my eyes 😝

File: .github/ISSUE_TEMPLATE/bug.yaml (hunk 6 above)
Comment (StyleShit) [reply .github/ISSUE_TEMPLATE/bug.yaml:1]: Yeah, I just remembered our YAMLs to be `.yml`
Never mind, if it works it works

**Review by StyleShit — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/custom.yaml [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+  - type: checkboxes
+    id: support_question
+    attributes:
+      label: "❓ Support Question"
+      options:
+        - label: "Please create GitHub issues only for bugs and feature requests. GitHub issues ARE NOT FOR SUPPORT!"
+        - label: "If you have questions or need general support, please use:  https://wordpress.org/support/plugin/elementor"
+        - label: "For help and support from the Elementor community, see: https://www.facebook.com/groups/Elementors/"
+        - label: "To read more about Elementor, check out our documentation: https://docs.elementor.com"
+        - label: "Developers docs are located at: https://developers.elementor.com/"
```
Comment (matipojo) [inline .github/ISSUE_TEMPLATE/custom.yaml:14]:
```suggestion
        - "Please create GitHub issues only for bugs and feature requests. GitHub issues ARE NOT FOR SUPPORT!"
        - "If you have questions or need general support, please use:  https://wordpress.org/support/plugin/elementor"
        - "For help and support from the Elementor community, see: https://www.facebook.com/groups/Elementors/"
        - "To read more about Elementor, check out our documentation: https://docs.elementor.com"
        - "Developers docs are located at: https://developers.elementor.com/"
```

**Review by matipojo — CHANGES_REQUESTED** [review]
The Custom template content should not be changed

File: .github/ISSUE_TEMPLATE/custom.yaml (hunk 10 above)
Comment (ItzhakAvm) [reply .github/ISSUE_TEMPLATE/custom.yaml:14]: I tried, didn't work.

**Review by matipojo — CHANGES_REQUESTED** [review]
Please start and target to `master`

**Pull request merged**